example/example.go

```diff
@@ -0,0 +1,31 @@
+package example
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/a-h/debounce"
+	"github.com/stianeikeland/go-rpio/v4"
+)
+
+func main() {
+	err := rpio.Open()
+	if err != nil {
+		log.Fatalln("Could not open GPIOs")
+	}
+	onClick := func() {
+		fmt.Print("Clicked...")
+	}
+	normallyClosed := true
+	sw := debounce.Button(onClick, normallyClosed)
+	// Set your GPIO here
+	gpio := rpio.Pin(0)
+	rpio.PinMode(gpio, rpio.Input)
+	getgpio := func() {
+		for {
+			sw.SetState(rpio.ReadPin(gpio) == rpio.High)
+		}
+	}
+	go getgpio()
```

Comment:
Here, starting the goroutine with `go` causes the loop to happen in the background. This means that the program then immediately carries on and exits instead of waiting.

```suggestion
	for {
		sw.SetState(rpio.ReadPin(gpio) == rpio.High)
	}
```